### DISCUSSION TITLE
Standardeinstellungen auf LS Basis

### DISCUSSION THREAD

**Discussion by Lahmi** [Mogli12/KeyboardSteer#747] [Previous version]
Hallo und vielen Dank für den tollen Mod und der vielen Arbeit damit.

Ist es möglich wenn dieser Mod genutzt wird diesen so zu programmieren, das dieser bei Aktivierung erst die Standarteinstellungen von LS läd und man anschließend seine Konfiguration einstellen muß?

Ich kenne viele Benutzer die den Mod wegen GPS nutzen aber sie möchten keine Fahrzeugdaten ändern. Deswegen würde man sich diese Einstellung wünschen.

Vielen Dank

Gruß Lahmi

**Comment by newbie-near-cologne** [comment]
Hallo, im Bezug auf meinem Post, [Link](https://github.com/Mogli12/VehicleControlAddon/issues/594#issuecomment-878087839), würde ich es auch begrüßen wenn standardmäßig die Giants Einstellungen verwendet werden. Da auf einem Dedi Server, zumindest bei uns, weder die globalen Einstellungen bei neu gekauften Fahrzeugen übernommen werden, und das speichern der eigenen Einstellungen pro Fahrzeug nur während man eingeloggt ist funktioniert, wäre es besser er nimmt immer die Standard Einstellungen, solange man nichts anderes einstellt.

Gruß Andre

**Comment by Mogli12** [comment]
In FS22 sollte jetzt alles aus sein.
